data/scripts/pkmn_center_nurse.inc

```diff
@@ -36,6 +36,13 @@ EventScript_PkmnCenterNurse_TakeAndHealPkmn::
 	applymovement VAR_0x800B, Movement_PkmnCenterNurse_Turn @ Changed from Common_Movement_WalkInPlaceFasterLeft to force the follower to enter their Poké Ball
 	waitmovement 0
 	dofieldeffect FLDEFF_POKECENTER_HEAL
+.if OW_UNION_DISABLE_CHECK == FALSE
+.if OW_FLAG_MOVE_UNION_ROOM_CHECK != 0
```

Comment:
I'm not sure, but is it possible to use `&&` in an `.if` statement? If so, that would clean this up considerably.

Reply:
[`0e9ea43` (#5448)](https://github.com/rh-hideout/pokeemerald-expansion/pull/5448/commits/0e9ea43a912e271ba87c1c6dccda761356a65812)